README.md

```diff
@@ -1,2 +1,4 @@
 # ttt-online
 TicTacToe Online
+
+To generate a secret key, use Fernet.generate_key()
```

Comment:
Create README.md file at server folder, and there you can provide description of server side of application, but not in the main description